src/allotropy/parsers/bmg_mars/bmg_mars_structure.py

```diff
@@ -67,19 +70,24 @@ def create(data: SeriesData, header_content: str) -> Header:
         read_type = valid_value_or_raise("read type", read_types, ReadType)
 
         # Get wavelengths from RawData line
-        raw_wavelengths = assert_not_none(
-            re.search(
-                r"Raw Data \((?P<wavelength1>\d+)(?:/)?(?P<wavelength2>\d+)?(?:\))",
-                header_content,
-            ),
-            msg="Wavelengths not found in input file.",
+        raw_data_match = assert_not_none(
+            re.search(r"Raw Data \(.*?\)", header_content),
+            msg="Raw Data line not found in input file.",
         )
+
+        wavelength = None
         excitation_wavelength = None
-        if raw_wavelengths.group("wavelength2"):
-            wavelength = float(raw_wavelengths.group("wavelength2"))
-            excitation_wavelength = float(raw_wavelengths.group("wavelength1"))
+
+        # Formats: "Raw Data (Ex/Em)", "Raw Data (Em)", "Raw Data (No filter)"
+        filter_info = raw_data_match.group(0).split("(")[1].rstrip(")")
```

Comment:
Because we are loosening the regex match here a bit, we open up the chance for an ill-formatted input to cause an error here.

Can you wrap the whole wavelength parsing block in a try/except here and raise an `AllotropeConversionError` with a message like: f"Failed to parse wavelength from Raw Data line: {raw_data_match.group(0)}"